src/test/java/htsjdk/variant/vcf/VCFHeaderLineTranslatorUnitTest.java

```diff
@@ -0,0 +1,64 @@
+        Assert.assertEquals(values2.get("Description"), "ANNOTATION != \"NA\" || ANNOTATION <= 0.01");
+
+        // test escaped quotes and an escaped backslash
+        final Map<String,String> values3 = VCFHeaderLineTranslator.parseLine(VCFHeaderVersion.VCF4_2, "<ID=ANNOTATION,Description=\"ANNOTATION \\\\= \\\"NA\\\" || ANNOTATION <= 0.01\">", null);
+        Assert.assertEquals(values3.size(), 2);
+        Assert.assertEquals(values3.get("ID"), "ANNOTATION");
+        Assert.assertEquals(values3.get("Description"), "ANNOTATION \\= \"NA\" || ANNOTATION <= 0.01");
+
+        // test a header line with two value tags, one with an escaped backslash and two escaped quotes, one with an escaped quote
+        final Map<String,String> values4 = VCFHeaderLineTranslator.parseLine(VCFHeaderVersion.VCF4_2, "<ID=ANNOTATION,Description=\"ANNOTATION \\\\= \\\"NA\\\" || ANNOTATION <= 0.01\", Description2=\"foo\\\"bar\">", null);
+        Assert.assertEquals(values4.size(), 3);
+        Assert.assertEquals(values4.get("ID"), "ANNOTATION");
+        Assert.assertEquals(values4.get("Description"), "ANNOTATION \\= \"NA\" || ANNOTATION <= 0.01");
+        Assert.assertEquals(values4.get("Description2"), "foo\"bar");
+
+        // test a line with a backslash that appears before something other than a quote or backslash
+        final Map<String,String> values5 = VCFHeaderLineTranslator.parseLine(VCFHeaderVersion.VCF4_2, "<ID=ANNOTATION,Description=\"ANNOTATION \\n with a newline in it\">", null);
+        Assert.assertEquals(values5.size(), 2);
+        Assert.assertEquals(values5.get("ID"), "ANNOTATION");
+        Assert.assertEquals(values5.get("Description"), "ANNOTATION \\n with a newline in it");
+
+        // test with an unclosed quote
+        try {
+            final Map<String, String> values6 = VCFHeaderLineTranslator.parseLine(VCFHeaderVersion.VCF4_2, "<ID=ANNOTATION,Description=\"ANNOTATION \\n with a newline in it>", null);
+            Assert.fail("Should have thrown a TribbleException for having an unclosed quote in the description line");
+        } catch (TribbleException.InvalidHeader e) {
+            Assert.assertEquals("Your input file has a malformed header: Unclosed quote in header line value <ID=ANNOTATION,Description=\"ANNOTATION \\n with a newline in it>", e.getMessage());
+        }
+
+        // test with an escaped quote at the end
+        try {
+            final Map<String, String> values7 = VCFHeaderLineTranslator.parseLine(VCFHeaderVersion.VCF4_2, "<ID=ANNOTATION,Description=\"ANNOTATION \\n with a newline in it\\\">", null);
+            Assert.fail("Should have thrown a TribbleException for having an unclosed quote in the description line");
+        } catch (TribbleException.InvalidHeader e) {
+            Assert.assertEquals("Your input file has a malformed header: Unclosed quote in header line value <ID=ANNOTATION,Description=\"ANNOTATION \\n with a newline in it\\\">", e.getMessage());
```

Comment:
Same here -- don't check the contents of the exception message in the test, just make sure `TribbleException.InvalidHeader` is thrown.